GraphQL/src/mutation_resolvers.py

```diff
@@ -0,0 +1,64 @@
+import asyncio
+import httpx
+
+
+async def add_user(username: str, email: str, password: str):
+    from src.schema import AddUserResponse, User
+
+    async with httpx.AsyncClient() as client:
+        try:
```

Comment:
This try;except is huge, please reduce its size to include only what is really meant to be tried